static/src/js/components/BoundingRectangleField/BoundingRectangleField.scss

```diff
@@ -21,5 +21,10 @@
     width: 200px;  
     text-align: center;
   }
-}
 
+  input::-webkit-outer-spin-button,
```

Comment:
This change should be done globally so all of the number input field do not have the scroll wheel behavior

Reply:
The AC only has it being removed in this particular area >> https://bugs.earthdata.nasa.gov/browse/MMT-3624. I can't remember, did we verbally agree that we don't want them anywhere? If so, I'll update the AC